podcasts/AutoplayHelper.swift

```diff
@@ -47,7 +47,10 @@ class AutoplayHelper {
             case .starred:
                 return .starred
             case .uuid(let uuid):
-                if let filter = DataManager.sharedManager.findFilter(uuid: uuid) {
+                guard uuid.isEmpty == false else {
```

Comment:
Nit: 
```
if uuid.isEmpty {
   return nil
}
```

Reply:
I kind of prefer the guard because I'm always worried the early return isn't as clear. Would `guard !uuid.isEmpty` be preferable?